AdvancedDLSupport.AOT/PregeneratedAssemblyBuilder.cs

```diff
@@ -176,11 +169,11 @@ public string Build([NotNull] string outputPath)
             var automaticInterfaces = new List<Type>();
             foreach (var sourceAssembly in SourceAssemblies)
             {
-                _log.Info($"Scanning {sourceAssembly.GetName().Name}...");
                 foreach (var automaticInterface in sourceAssembly.ExportedTypes.Where(t => t.HasCustomAttribute<AOTTypeAttribute>()))
                 {
                     automaticInterfaces.Add(automaticInterface);
-                    _log.Info($"Discovered {automaticInterface.Name}.");
+
+                    // TODO logging
```

Comment:
Why was the logging removed?

Reply:
MSBuild can't use NLog. Mind, I could have it so NLog can be used only if available. Will fix now.

Reply:
What's the effect if used through an MSBuild task? A crash, nothing happening...?

Reply:
The user will just never see the logs I think.

Reply:
Might as well not remove it, then, since it'd work when running from the command line.

Reply:
Fixed.

Reply:
That being said, is it possible to inject the logging instance used in the task?

Reply:
Yes, but it'd require us referencing MSBuild from the console application.

Reply:
Let's leave it as is for now, then. It's not critical.